src/Model/Setting/BookkeepingSetting.php

```diff
@@ -0,0 +1,549 @@
+    {
+        $new = clone $this;
+        $new->incomeAccountForVat0 = $incomeAccountForVat0;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getReversedVatAccount(): string
+    {
+        return $this->reversedVatAccount;
+    }
+
+    /**
+     * @param string $reversedVatAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withReversedVatAccount(string $reversedVatAccount)
+    {
+        $new = clone $this;
+        $new->reversedVatAccount = $reversedVatAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getVatAccountForVat25(): string
+    {
+        return $this->vatAccountForVat25;
+    }
+
+    /**
+     * @param string $vatAccountForVat25
+     *
+     * @return BookkeepingSetting
+     */
+    public function withVatAccountForVat25(string $vatAccountForVat25)
+    {
+        $new = clone $this;
+        $new->vatAccountForVat25 = $vatAccountForVat25;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getVatAccountForVat12(): string
+    {
+        return $this->vatAccountForVat12;
+    }
+
+    /**
+     * @param string $vatAccountForVat12
+     *
+     * @return BookkeepingSetting
+     */
+    public function withVatAccountForVat12(string $vatAccountForVat12)
+    {
+        $new = clone $this;
+        $new->vatAccountForVat12 = $vatAccountForVat12;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getVatAccountForVat6(): string
+    {
+        return $this->vatAccountForVat6;
+    }
+
+    /**
+     * @param string $vatAccountForVat6
+     *
+     * @return BookkeepingSetting
+     */
+    public function withVatAccountForVat6(string $vatAccountForVat6)
+    {
+        $new = clone $this;
+        $new->vatAccountForVat6 = $vatAccountForVat6;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getAccountReceivableAccount(): string
+    {
+        return $this->accountReceivableAccount;
+    }
+
+    /**
+     * @param string $accountReceivableAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withAccountReceivableAccount(string $accountReceivableAccount)
+    {
+        $new = clone $this;
+        $new->accountReceivableAccount = $accountReceivableAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getClientFundsAccount(): string
+    {
+        return $this->clientFundsAccount;
+    }
+
+    /**
+     * @param string $clientFundsAccount
+     *
+     * @return BookkeepingSetting
+     */
+    public function withClientFundsAccount(string $clientFundsAccount)
+    {
+        $new = clone $this;
+        $new->clientFundsAccount = $clientFundsAccount;
+
+        return $new;
+    }
+
+    /**
+     * @return string
+     */
+    public function getBankingAccount(): string
```

Comment:
Will this **always** return a string? Can `$this->bankingAccount` never be null?